WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -76,13 +77,15 @@ private enum Trigger
 
 	private readonly MusicStatusMessageViewModel _finishedMessage = new() { Message = "No balance to coinjoin" };
 
-	private DateTimeOffset _autoStartTime;
+	private TimeSpan _autoStartTime;
 	private DateTimeOffset _countDownStarted;
 
 	public CoinJoinStateViewModel(WalletViewModel walletVm, IObservable<Unit> balanceChanged)
 	{
 		_elapsedTime = "";
 		_remainingTime = "";
+		_countDownStarted = DateTimeOffset.Now;
+		_autoStartTime = TimeSpan.FromSeconds(Random.Shared.Next(1 * 60, 1 * 60));
```

Comment:
The original was 5-16 minutes. 

https://github.com/zkSNACKs/WalletWasabi/pull/7411/commits/35de321657bb93c42a5fbc2cbb112dd97e69d58a

Reply:
oops accidentally committed that.. when testing.

Reply:
fixed.